playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -29,6 +29,10 @@
       {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }} --force --delete-local-data
     delegate_to: "{{ groups.oo_first_master.0 }}"
 
+  - name: Pause for pod rescheduling during node ugprade
+    pause:
+      seconds: "{{ openshift_node_upgrade_pause | default(30) | int }}"
+
```

Comment:
I wasn't entirely sure if the problem is because we start killing the node before pods have been successfully started elsewhere or if it was that we move on to the next node. I figured by placing it immediately after the drain operation it would address both cases.